logging/src/jsMain/kotlin/StackTraceTagGenerator.kt

```diff
@@ -13,6 +13,7 @@ internal object StackTraceTagGenerator : TagGenerator {
     private val stackTraceLineMatcher = Regex("""^\s{4}at (\w+)\..*$""")
 
     override fun getTag(): String {
+        // FIXME: This should implement support for [HideFromStackTrace] instead of hard-coding a known depth
```

Comment:
We should probably get this working before we do a release, right?
If so, it is likely deserving of a ticket so we can mark it with a milestone (to make sure we don't miss it).

Reply:
Right now this _can't_ be fixed, because the JS stack trace is 1) slightly different for different browsers, and 2) does not provide full package names on _any_ of them (are packages even a concept in JS?). Regardless, the net effect is that we need support from the Kotlin compiler.

After this is merged I'll file an issue so we can at least track it. We can also discuss workarounds in that as a thread. Either way, I don't think this is a release blocker.